apis/server/container_bridge.go

```diff
@@ -10,13 +10,26 @@ import (
 	"time"
 
 	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/daemon/mgr"
 	"github.com/alibaba/pouch/pkg/httputils"
 
 	"github.com/gorilla/mux"
 )
 
-// TODO
 func (s *Server) removeContainers(ctx context.Context, resp http.ResponseWriter, req *http.Request) error {
+	name := mux.Vars(req)["name"]
+
+	option := &mgr.ContainerRemoveOption{
+		Force:  httputils.BoolValue(req, "force"),
+		Volume: httputils.BoolValue(req, "v"),
```

Comment:
I do not think we have implemented remove flags `v` and `l`. 
Can we remove both two, or mark this and implement them in the future?

If the answer is latter one, I think we had better track this in an issue or mark that in code comments.